src/main/java/step2/StringAdderCalculator.java

```diff
@@ -0,0 +1,68 @@
+package step2;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import step2.ArgumentResolver.CustomResolver;
+import step2.ArgumentResolver.DefaultResolver;
+import step2.ArgumentResolver.EmptyStringResolver;
+import step2.ArgumentResolver.InputSingleNumberResolver;
+import step2.ArgumentResolver.Resolver;
+import step2.validator.NotNumberValidator;
+import step2.validator.PositiveNumberValidator;
+import step2.validator.Validator;
+
+public class StringAdderCalculator {
+
+    private static final ArrayList<Resolver> resolvers = new ArrayList<Resolver>();
+    private static final ArrayList<Validator> validators = new ArrayList<Validator>();
```

Comment:
타입 아규먼트는 생략할 수 있습니다.
```suggestion
    private static final ArrayList<Resolver> resolvers = new ArrayList<>();
    private static final ArrayList<Validator> validators = new ArrayList<>();
```